deployment/GI05MOAS-GL365_Deploy.csv

```diff
@@ -1,5 +1,9 @@
 CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes
-AR46,,,,GI05MOAS-GL365-00-ENG000000,3,1,2020-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00365,CGCON-GLDROO-00365,59,-39,,1000,1000,
-AR46,,,,GI05MOAS-GL365-01-FLORDM000,3,1,2020-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00365,CGINS-FLORDM-03124,59,-39,,1000,1000,
-AR46,,,,GI05MOAS-GL365-02-DOSTAM000,3,1,2020-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00365,CGINS-DOSTAM-00281,59,-39,,1000,1000,
-AR46,,,,GI05MOAS-GL365-04-CTDGVM000,3,1,2020-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00365,CGINS-CTDGVM-09061,59,-39,,1000,1000,
\ No newline at end of file
+AR46,,AR46,,GI05MOAS-GL365-00-ENG000000,3,1,2020-08-15T18:18:00,2020-08-16T00:49:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00365,CGCON-GLDROO-00365,59.9267,-39.300067,,1000,1000,GPS issue
+AR46,,AR46,,GI05MOAS-GL365-01-FLORDM000,3,1,2020-08-15T18:18:00,2020-08-16T00:49:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00365,CGINS-FLORDM-03124,59.9267,-39.300067,,1000,1000,GPS issue
+AR46,,AR46,,GI05MOAS-GL365-02-DOSTAM000,3,1,2020-08-15T18:18:00,2020-08-16T00:49:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00365,CGINS-DOSTAM-00281,59.9267,-39.300067,,1000,1000,GPS issue
+AR46,,AR46,,GI05MOAS-GL365-04-CTDGVM000,3,1,2020-08-15T18:18:00,2020-08-16T00:49:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00365,CGINS-CTDGVM-09061,59.9267,-39.300067,,1000,1000,GPS issue

```

Comment:
Deploy time should be 16:18:00.

Reply:
Indeed it should!  Those 6s and 8s...they often look the same to me...fixed